xapian-applications/omega/index_file.cc

```diff
@@ -551,44 +552,45 @@ bool index_update_entry(const string & urlterm,DirectoryIterator &d){
 
 void
 index_mimetype(const string & file, const string & urlterm, const string & url,
-	       const string & ext, string &mimetype, DirectoryIterator &d,
-	       Xapian::Document & newdocument,
-	       string record)
+	       const string & ext, string &mimetype, string path_term,
+               DirectoryIterator &d, string record)
 {
     string context(file, root.size(), string::npos);
 
     time_t last_altered = use_ctime ? d.get_ctime() : d.get_mtime();
 
     Xapian::docid did = 0;
     if (index_check_existing(urlterm, last_altered, did))
-	return;
```

Comment:
This (and a lot of the later line changes) seem to be whitespace only — suggesting your editor has reformatted them for some reason. These changes probably shouldn't be part of this commit. (If the formatting is incorrect, a separate reformatting commit is much clearer.)

Reply:
I will try to avoid this kind of changes.